wordpress-mobile/WordPress-iOS
# Enable New Reader Detail

Merges the reader detail updates into develop

### Screenshots
| No Featured Image | With Featured Image |
|:---:|:---:|
|![simulator_screenshot_CFF03FA8-CB61-4422-BA4F-1FD942293EDF](https://user-images.githubusercontent.com/793774/96041988-8a2f5700-0e21-11eb-9efc-c52c12c4581a.png)|![simulator_screenshot_357B3346-5FB6-4FD6-92D2-EB8652E34B62](https://user-images.githubusercontent.com/793774/96042068-adf29d00-0e21-11eb-8fc9-2a41b09cca62.png)|


### To test:
Should be good If the build is 🟢 

You can also test the Reader Detail by doing:

1. Launch the app
2. Tap on the Reader
3. Tap on Discover
4. Locate any post
5. Tap on it

### PR submission checklist:
